### PR TITLE
Adds search by users full name for visit requests (#269)

### LINKED ISSUE #269: Search by user full name at admin/visit_requests#index when event is live
Search by visitor's full_name at:
/admin/event/:id/visit_requests

### PR DESCRIPTION
**Pull request by kushniryb**
Fixed #269

### CODE REVIEW

File: app/modules/searchable/configuration/friend.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module Friend
       FIELDS = %i(name)
 
       define_searchable do
-        pg_search_scope :search, against: FIELDS, using: TSEARCHABLE_WITH_PREFIX
+        pg_search_scope :search, against: FIELDS, using: TS_P
```
Comment (denys-medynskyi) [inline app/modules/searchable/configuration/friend.rb]: @kushniryb I like the long name more. It is more clear.

**Review by sveredyuk — APPROVED** [review]

File: app/views/admin/visit_requests/index.slim [hunk 2]
```diff
@@ -1,33 +1,37 @@
-h2 = event.title
-= event_status_label(event)
+.ui.stackable.grid
```
Comment (denys-medynskyi) [inline app/views/admin/visit_requests/index.slim:1]: when we do changes in view - should we add screenshots ? what do you think @kushniryb ?

**Review by denys-medynskyi — APPROVED** [review]

File: app/views/admin/visit_requests/index.slim (hunk 2 above)
Comment (kushniryb) [reply app/views/admin/visit_requests/index.slim:1]: vr_view

Comment (kushniryb) [reply app/views/admin/visit_requests/index.slim:1]: Here you go @denysmedynskyi

**Pull request merged**